connectors/sources/sharepoint_server.py

```diff
@@ -689,15 +686,14 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: dictionary containing meta-data of the SharePoint objects.
         """
-
         server_relative_url = []
 
-        for collection in self.sharepoint_client.site_collections_path:
-            server_relative_url.append(collection)
+        for collection in self.sharepoint_client.site_collections:
+            server_relative_url.append(f"{collection}")
```

Comment:
What does this change? Collection is a string, why stringify again?

Reply:
To be fixed.